{"name": "test_login_failed_invalid_data[case1]", "status": "passed", "parameters": [{"name": "case", "value": "{'user': '13252254992', 'passwd': '', 'check': '请输入密码'}"}], "start": 1636516636490, "stop": 1636516637659, "uuid": "7418ebaf-4538-44d3-8366-be50dc5fc01c", "historyId": "1352fcf7b98c3a086b04449cfbf3bcb3", "testCaseId": "98e067a681f547a2e44e78c948720b9e", "fullName": "TestCases.test_login.TestLogin#test_login_failed_invalid_data", "labels": [{"name": "epic", "value": "[epic]盛杰运营后台"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh')"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-OTOI92D"}, {"name": "thread", "value": "9592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}